packages/deployed-backend-client/src/deployed_backend_client.ts

```diff
@@ -141,9 +141,7 @@ export class DefaultDeployedBackendClient implements DeployedBackendClient {
       nextToken = listStacksResponse.nextToken;
 
       if (stackMetadata.length !== 0) {
-        yield {
-          backends: stackMetadata,
-        };
+        yield stackMetadata;
       }
```

Comment:
What I meant was yield individual metadata objects. eg
```ts
for (const entry of stackMetadata) {
  yield entry
}
```

This prevents the caller from having to do a nested loop to get the entries